community/modules/compute/schedmd-slurm-gcp-v6-nodeset-tpu/main.tf

```diff
@@ -34,9 +34,8 @@ locals {
     docker_image = var.docker_image
 
     enable_public_ip = !var.disable_public_ips
-    # TODO: rename to subnetwork_self_link, requires changes to the scripts
```

Comment:
pls don't remove TODO

Reply:
Bad merge.  Thanks for catching this.  Added it back.